src/pydata_sphinx_theme/assets/styles/components/_breadcrumbs.scss

```
@@ -0,0 +1,38 @@
+/**
+ * Breadcrumbs for parent pages meant for the article header
+ */
+ul.bd-breadcrumbs {
+  list-style: none;
+  padding-left: 0;
+  display: flex;
+  flex-wrap: wrap;
+
+  // Font size slightly smaller to avoid cluttering up space too much
+  font-size: 0.8em;
```

any reason you preferred the relative `em` over `rem` ? I would suggest to stick to the absolute `rem` as much as possible to avoid issues with nested elements. 

```suggestion
  font-size: 0.8rem;
```

---

in this case it was because I wanted it to be relative to the main font in the content page, but I think it's fine if we go with rem as well likely